packages/field/test-suites/FormatMixin.suite.js

```diff
@@ -0,0 +1,394 @@
+import { expect, fixture, html, aTimeout, defineCE, unsafeStatic } from '@open-wc/testing';
+import sinon from 'sinon';
+
+import { LitElement } from '@lion/core';
+import { Unparseable } from '@lion/validate';
+import { FormatMixin } from '../src/FormatMixin.js';
+
+function mimicUserInput(formControl, newViewValue) {
+  formControl.value = newViewValue; // eslint-disable-line no-param-reassign
+  formControl.inputElement.dispatchEvent(new CustomEvent('input', { bubbles: true }));
+}
+
+export function runFormatMixinSuite(customConfig) {
+  // TODO: Maybe remove suffix
+  const cfg = {
+    tagString: null,
+    modelValueType: String,
+    suffix: '',
+    ...customConfig,
+  };
+
+  /**
+   * Mocks a value for you based on the data type
+   * Optionally toggles you a different value
+   * for needing to mimic a value-change.
+   */
+  function generateValueBasedOnType(opts = {}) {
```

Comment:
Nice solution :)
Maybe we later elaborate on it in such a way that it doesn't know anything about our custom inputs.
That would mean it knows about platform types, but not about iban, email etc.